docs/source/general/settings.rst

```diff
@@ -180,3 +180,12 @@ The text could be anything but should have the format string operator ``{}`` for
 It is the sender email which is also to be configured in the SMTP settings.
 The default sender email is a common setting from the `Django core settings  <https://docs.djangoproject.com/en/2.1/ref/settings/#default-from-email>`_ under ``DEFAULT_FROM_EMAIL``.
 Currently, ``DEFAULT_FROM_EMAIL`` is set to to ``webmaster@localhost``.
+
+``DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
`~~~~` line is too long, it should equal the length of the text, eg:

```
``DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

some ReStructuredText parsers complain when this rule is not followed.

Reply:
I just changed this, @nemesisdesign